src/client/screens/CreateProjectScreen.tsx

```diff
@@ -82,6 +88,7 @@ const CreateProjectScreen = ({ regionConfigs }: StateProps) => {
         ...(chamber
           ? {
               numberOfDistricts: chamber.numberOfDistricts,
+              chamber: chamber || null,
               isCustom: false
             }
           : { numberOfDistricts: null, isCustom: true })
```

Comment:
You need to explicitly null out `chamber` here or switching to a chamber & back to custom will still set a chamber ID.
```suggestion
          : { numberOfDistricts: null, isCustom: true, chamber: null })
```